src/Lecture1.hs

```diff
@@ -81,7 +84,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z =
+    let biggest = foldl max x [y, z]
```

Comment:
Clever usage of `foldl` 🤓 

In this case, a repeated usage of `max` is fine as well and slightly more efficient

```suggestion
    let biggest = max x (max y z)
```